proto/user-service.proto

```diff
@@ -52,6 +64,20 @@ message UpdateUserResponse {
     User user = 1;
 }
 
+message ValidateTokenResponse {
+    bool valid = 1;
+}
+
+message LoginRequest {
+    string username = 1;
```

Comment:
LoginRequest only requires a Username & password? 

AuthRequest needs a token & username to verify that the token comes from the right user. Or that was the idea behind it.